components/datatypes/industry-verticals/form-applications.schema.json

```diff
@@ -14,54 +14,54 @@
   "definitions": {
     "form-applications": {
       "properties": {
-        "xdm:formApplicationApproved": {
+        "xdm:approved": {
           "title": "Form Application Approved",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "Captured when a visitor application has been approved."
         },
-        "xdm:formApplicationDeclined": {
+        "xdm:declined": {
           "title": "Form Application Declined",
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "Captured when a visitor application has been declined."
         },
-        "xdm:formApplicationFee": {
-          "title": "Form Application Fee",
+        "xdm:applicationFee": {
```

Comment:
Given the changes you did here, should this be just "fee" ?

Reply:
I purposely left it as "application fee", because that made more sense in context rather than just "fee".